.github/workflows/stale.yml

```diff
@@ -0,0 +1,44 @@
+name: Close Stale Issues
+on:
+  schedule:
+    # Run at midnight every night
+    - cron: '0 0 * * *'
```

Comment:
Maybe let's pick a different, more random time, so that github doesn't have a midnight load spike? I personally like 1:24am

Reply:
powers of two?

Reply:
lol yes, but 10:24 seemed too early